player/player.go

```diff
@@ -29,6 +31,7 @@ func NewPlayer(hotCache *HotCache, lbrynetAddress string) *Player {
 	return &Player{
 		lbrynetClient: ljsonrpc.NewClient(lbrynetAddress),
 		blobSource:    hotCache,
+		resolveCache:  ccache.New(ccache.Configure().MaxSize(1000)),
```

Comment:
I think it doesn't hurt to have 10k instead. we're still resolving all thumbnails from all the sections, possibly the videos and a few random things. We can then monitor how good it is but I feel like 1k is too little